README.md

```diff
@@ -8,4 +8,8 @@ This is a brand new repo. I'm hoping that this will be a useful exercise for ear
 
 ## Contributing
 
+# We love 😘💕 your contributions.
+
+## So..why waiting?
```

Comment:
Jakaya thanks for contributing!

Please remove line 13 and 12. 

It would be more native english to say `Why wait?`, but we don't need to say this overall.

Thank you for contributing and I like the emojis. :D